packages/app/src/components/transactions/infoTable/GeneralInfo.vue

```diff
@@ -353,6 +366,9 @@ const gasUsedPercent = computed(() => {
   .transaction-status-value {
     @apply py-2;
   }
+  .transaction-error-value {
```

Comment:
`.transaction-error-value` and `.transaction-reason-value` are identical. To avoid duplication, let's leave just `.transaction-error-value` and use for both error and error reason.